tools/internal/autoupdate/config.go

```diff
@@ -77,10 +86,18 @@ func (entry ConfigEntry) Validate() error {
 		return errors.New("must specify Name")
 	}
 
-	if entry.GithubLatestRelease == nil {
+	if entry.GithubLatestRelease == nil && entry.HelmLatest == nil {
 		return errors.New("must specify an autoupdate strategy")
-	} else if err := entry.GithubLatestRelease.Validate(); err != nil {
-		return fmt.Errorf("GithubLatestRelease failed validation: %w", err)
+	} else if entry.GithubLatestRelease != nil && entry.HelmLatest != nil {
```

Comment:
I think it's clearer if we just have multiple `if` instead of `else if` since there's no chance of entering more than one if anyways